src/amo/components/AddonsCard/index.js

```diff
@@ -12,7 +12,7 @@ import type {
   DeleteAddonNoteFunc,
   RemoveCollectionAddonFunc,
   SaveAddonNoteFunc,
-} from 'amo/components/Collection';
+} from 'amo/components/pages/Collection';
```

Comment:
I'd prefer `amo/pages/Collection`. I get that they are technically components but I just don't want to go digging for them -- there are enough component sub-directories as it is. I'd rather see it moved up a level.

Reply:
Let's do this!